app/assets/src/api/index.js

```diff
@@ -46,11 +60,15 @@ const getAlignmentData = (sampleId, alignmentQuery, pipelineVersion) =>
     `/samples/${sampleId}/alignment_viz/${alignmentQuery}.json?pipeline_version=${pipelineVersion}`
   );
 
+const deleteSample = id => deleteWithCSRF(`/samples/${id}.json`);
+
 export {
   getSampleMetadata,
   saveSampleMetadata,
   getMetadataTypes,
   saveSampleName,
   saveSampleNotes,
-  getAlignmentData
+  getAlignmentData,
+  getURLParamString,
+  deleteSample
```

Comment:
Should we start splitting some of these (metadata & sample)?

Reply:
The metadata is sample metadata, so not sure if that distinction makes sense. I think we can wait a little longer, but should definitely split this eventually.

Reply:
The logic sounds good. I would still put all in `api/sample.js` and leave the generic methods here.
Can be in a future follow up.